config-files/adf-templates/excel-sheet-ingestion.json

```diff
@@ -0,0 +1,351 @@
+                        },
+                        "formatSettings": {
+                            "type": "JsonReadSettings"
+                        }
+                    },
+                    "dataset": {
+                        "referenceName": "configFile",
+                        "type": "DatasetReference",
+                        "parameters": {
+                            "configFile": {
+                                "value": "@pipeline().parameters.configFile",
+                                "type": "Expression"
+                            },
+                            "configPath": {
+                                "value": "@pipeline().parameters.configPath",
+                                "type": "Expression"
+                            },
+                            "configFS": {
+                                "value": "@pipeline().parameters.configFS",
+                                "type": "Expression"
+                            }
+                        }
+                    }
+                }
+            },
+            {
+                "name": "set fileSystem",
+                "type": "SetVariable",
+                "dependsOn": [
+                    {
+                        "activity": "configFile lookup",
+                        "dependencyConditions": [
+                            "Succeeded"
+                        ]
+                    }
+                ],
+                "userProperties": [],
+                "typeProperties": {
+                    "variableName": "fileSystem",
+                    "value": {
+                        "value": "@activity('configFile lookup').output.firstRow.pipeline.adl_file_system",
+                        "type": "Expression"
+                    }
+                }
+            },
+            {
+                "name": "set sink file",
+                "type": "SetVariable",
+                "dependsOn": [
+                    {
+                        "activity": "set source file",
+                        "dependencyConditions": [
+                            "Succeeded"
+                        ]
+                    },
+                    {
+                        "activity": "set source sheet",
+                        "dependencyConditions": [
+                            "Succeeded"
+                        ]
+                    }
+                ],
+                "userProperties": [],
+                "typeProperties": {
+                    "variableName": "sinkFile",
+                    "value": {
+                        "value": "@activity('configFile lookup').output.firstRow.pipeline.raw.sink_file",
+                        "type": "Expression"
+                    }
+                }
+            },
+            {
+                "name": "copy excel sheet to csv",
+                "description": "Copies the current file from ADL2 raw (sharepoint) to ADL2 raw time-stamped folder (YYYY-MM-DD)",
```

Comment:
Can we make this more generic. delete ref to sharepoint

Reply:
deleted reference to SharePoint in commit "update excel-sheet-ingestion"